### PR TITLE
Frontend Logout

### PR DESCRIPTION
**Pull request by Muartin**

### CODE REVIEW

File: Frontend/src/styles/Header.css [hunk 1]
```diff
@@ -51,4 +51,37 @@ nav img {
...
+}
+
+.dropdown-content a {
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+}
+
+.dropdown-content a:hover {
```
Comment (pokeblokdude) [inline Frontend/src/styles/Header.css:81]: Should really add `cursor: pointer;` to this.

**Review by pokeblokdude — COMMENTED** [review]
Also maybe make the shadow from the header go on top of the dropdown, so it looks like the dropdown is "below" the header? I think that would fit with the hierarchy style we decided on for the listing card as well.

File: Frontend/src/styles/Header.css (hunk 1 above)
Comment (Muartin) [reply Frontend/src/styles/Header.css:81]: Changed it so that cursor changes to pointer on hover over logout. Pushed it to the branch, hopefully it updated.

Comment (pokeblokdude) [reply Frontend/src/styles/Header.css:81]: It doesn't show any new commits

**Review by lhamdy — APPROVED** [review]

File: Frontend/src/components/JobListings.js [hunk 2]
```diff
@@ -52,15 +52,6 @@ const JobListings = ({ user, setPopupOpen, searchInput = '' }) => {
 
   return (
     <div className='job-listings-container'>
-      <p id='num-listings'>
```
Comment (MiniShark27) [inline Frontend/src/components/JobListings.js:55]: Out of curiosity, why was this removed? Wouldn't it be useful to know how many results there were?

Comment (pokeblokdude) [reply Frontend/src/components/JobListings.js:55]: That is a good question for @Muartin

Comment (pokeblokdude) [reply Frontend/src/components/JobListings.js:55]: That is a good question

**Review by MiniShark27 — APPROVED** [review]
I'm not on the Frontend... but the code looks good 🤷 . Here's my approval.

P.S. We should really get the frontend people to approve frontend PRs...

File: Frontend/src/components/ListingCard.js [hunk 3]
```diff
@@ -14,8 +14,7 @@ const ListingCard = ({ setPopupOpen, listingObj }) => {
 
   const handleClick = () => {
     if (!hover) {
-      setPopupOpen(false);
-      history.goBack();
+      handleClose();
```
Comment (pokeblokdude) [inline Frontend/src/components/ListingCard.js]: Making this a separate function just makes it harder to read, imo. It's literally only 2 lines of code

**Review by Spirolife — APPROVED** [review]
o/
